<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-Q9R4LGA" failures="0" tests="7" name="api.test.UserTest" time="6.891" errors="1" timestamp="2023-07-18T12:03:45 WIB" skipped="0">
  <testcase classname="api.test.UserTest" name="USR001" time="3.541"/>
  <system-out/>
  <testcase classname="api.test.UserTest" name="USR002" time="0.385"/>
  <system-out/>
  <testcase classname="api.test.UserTest" name="USR003" time="1.525">
    <error message="
[public void api.test.UserTest.USR003(java.lang.String,java.lang.String,java.lang.String,java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: USR003([java.lang.String arg0, java.lang.String arg1, java.lang.String arg2, java.lang.String arg3])
Arguments: [(java.lang.String) Ryu,(java.lang.String) ryu.muhammad@gomail.co.id,(java.lang.String) male,(java.lang.String) active,(java.lang.String) ]" type="org.testng.internal.reflect.MethodMatcherException">
      <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
[public void api.test.UserTest.USR003(java.lang.String,java.lang.String,java.lang.String,java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: USR003([java.lang.String arg0, java.lang.String arg1, java.lang.String arg2, java.lang.String arg3])
Arguments: [(java.lang.String) Ryu,(java.lang.String) ryu.muhammad@gomail.co.id,(java.lang.String) male,(java.lang.String) active,(java.lang.String) ]
at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:46)
at org.testng.internal.Parameters.injectParameters(Parameters.java:918)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:37)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- USR003 -->
  <system-out/>
  <testcase classname="api.test.UserTest" name="USR005" time="0.384"/>
  <system-out/>
  <testcase classname="api.test.UserTest" name="USR006" time="0.348"/>
  <system-out/>
  <testcase classname="api.test.UserTest" name="USR007" time="0.358"/>
  <system-out/>
  <testcase classname="api.test.UserTest" name="USR004" time="0.350"/>
  <system-out/>
</testsuite> <!-- api.test.UserTest -->
